src/vs/workbench/services/backup/node/backupFileService.ts

```diff
@@ -64,6 +67,14 @@ export class BackupFilesModel implements IBackupFilesModel {
 		return true;
 	}
 
+	public getTextFiles(): string[] {
```

Comment:
@Tyriar this is some crazy string magic that is not very robust, instead I suggest to use the `URI` apis we already have: use `URI.parse()` to convert the resource string into a `URI` and then you have access to `fsPath` and scheme to find out what you want.